adapters/wagmi-connector/src/bloctoConnector.ts

```diff
@@ -37,13 +37,12 @@ type BloctoOptions = {
    * Custom RPC endpoint.
    */
   rpc?: string;
-}
+};
 
 class BloctoConnector extends Connector<BloctoProvider, BloctoOptions> {
-  readonly id = 'bloctoWallet';
-  readonly name = 'Blocto Wallet';
+  readonly id = 'blocto';
```

Comment:
pls assist me to replacing the corresponding keys in connectkit & web3modal.

https://github.com/blocto/blocto-sdk/blob/d6843a6ea75d902be393a73ab049934f3b4b53ed/adapters/wagmi-connector/src/bloctoWeb3ModalConfig.ts#L6

https://github.com/blocto/blocto-sdk/blob/fe900e8127c1bd3e104c764848436eeea4b8848c/adapters/connectkit-connector/src/constants/index.tsx#L7C1-L8C1